### PR TITLE
[#2920] Intro buttons translated

### PR DESCRIPTION
**Pull request by kardan**
- [ ] **Update release notes if necessary**

### CODE REVIEW

File: client/src/translations/es.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
   "authentication_issue": "Existe un problema verificando su identidad",
   "autodetect_column_types": "Detectar tipos de datos automáticamente",
   "average": "Promedio",
+  "back": "Espalda",
```
Comment (valllllll2000) [inline client/src/translations/es.json:25]: :smile: sorry but Espalda does mean Back but the body part :smile: here I would say something like "Atrás"

File: client/src/translations/fr.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
   "authentication_issue": "Un problème est survenu lors de la vérification de votre identité",
   "autodetect_column_types": "Detecter automatiquement le type de colonne",
   "average": "Moyenne",
+  "back": "Arrière",
```
Comment (valllllll2000) [inline client/src/translations/fr.json:25]:
```suggestion
  "back": "Revenir",
```

File: client/src/translations/fr.json [hunk 3]
```diff
@@ -153,6 +154,7 @@
   "geotiff": "GeoTIFF",
   "github": "Github",
   "google_drive": "Google Drive",
+  "got_it": "Je l'ai",
```
Comment (valllllll2000) [inline client/src/translations/fr.json:157]:
```suggestion
  "got_it": "J'ai compri",
```

File: client/src/translations/es.json (hunk 1 above)
Comment (valllllll2000) [inline client/src/translations/es.json:25]:
```suggestion
  "back": "Volver",
```

**Review by valllllll2000 — CHANGES_REQUESTED** [review]
See comments and suggestions

File: client/src/translations/es.json (hunk 1 above)
Comment (kardan) [reply client/src/translations/es.json:25]: :-)

**Pull request merged**